src/components/templates/Audience/Audience.tsx

```diff
@@ -46,6 +43,14 @@ interface ChatOutDataType {
   text: string;
 }
 
+// If you change this, make sure to also change it in Audience.scss's $seat-size
+const SEAT_SIZE = "var(--seat-size)";
+const SEAT_SIZE_MIN = "var(--seat-size-min)";
+
+const VIDEO_MIN_WIDTH_IN_SEATS = 8;
+// We should keep the 16/9 ratio
+const VIDEO_MIN_HEIGHT_IN_SEATS = (VIDEO_MIN_WIDTH_IN_SEATS * 9) / 16;
```

Comment:
I think this makes the intent of it slightly clearer.

```suggestion
const VIDEO_MIN_HEIGHT_IN_SEATS = VIDEO_MIN_WIDTH_IN_SEATS * (9 / 16);
```